src/c/config.h

```diff
@@ -16,6 +16,12 @@
 
 #include <stdatomic.h>
 
+#define EX_METRIC_EVSENT 0x1
```

Comment:
I think because all the metrics we are doing at the moment are simple counters, there's not much to be gained from compiling in/out. If we ever implement the more complicated statistics-enabled ones we could look into it some more?